modules/sdk-coin-avaxp/src/lib/transactionBuilder.ts

```diff
@@ -121,129 +115,18 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
-  /**
-   * Getter for know if build should sign
-   */
-  get hasSigner(): boolean {
-    return this._signer !== undefined && this._signer.length > 0;
-  }
+  // TODO(CR-1073) Implement:
+  //  fromImplementation
+  //  buildImplementation
+  //  signImplementation
+  //  get transaction
+  //  set transaction
+  //  validateRawTransaction
```

Comment:
Some of these methods are already implemented in the permissionlessAddValidatorTxBuilder, we can move them here